benefits/settings.py

```diff
@@ -70,9 +71,10 @@ def _filter_empty(ls):
 CSRF_COOKIE_HTTPONLY = True
 CSRF_TRUSTED_ORIGINS = _filter_empty(os.environ["DJANGO_TRUSTED_ORIGINS"].split(","))
 
-SESSION_COOKIE_SAMESITE = "Strict"
+SESSION_COOKIE_SAMESITE = "Lax"
```

Comment:
Can you say more about this change?

Reply:
Yeah there was a big Slack thread about this, will tag you in again.

TLDR; with `Strict` the user loses their Django session between leaving our app to sign in with Login.gov, and coming back into our app from the Login.gov redirect. This is because `Strict` disallows cookies from external domains and so the session cookie is lost.

`Lax` allows the cookie to travel with the user and be sent back to us by Login.gov, as long as the request is "safe" i.e. GET, which it is.

Reply:
Moving the comment into the code for longevity:

```suggestion
# With `Strict`, the user loses their Django session between leaving our app to sign in with Login.gov, and coming back into our app from the Login.gov redirect. This is because `Strict` disallows cookies from external domains and so the session cookie is lost.
#
# `Lax` allows the cookie to travel with the user and be sent back to us by Login.gov, as long as the request is "safe" i.e. GET, which it is.
SESSION_COOKIE_SAMESITE = "Lax"
```

Reply:
See [842540b](https://github.com/cal-itp/benefits/pull/414/commits/842540b57e93f3c12c68723328afd478f4de5b39)